src/main/pages/che-6/user-guide/version-control.adoc

```diff
@@ -67,124 +97,127 @@ CHE_OAUTH_GITHUB_TOKENURI=https://github.com/login/oauth/access_token
-== Git Operations
+== Git operations
 
-*Commit*
+=== Commit
 
-Commit your changes at `Git` -> `Commit...` main menu or call it with `Alt + C` shortcut.
+Commit your changes using the *Git* > *Commit* menu options or use the `Alt + C` keyboard shortcut.
 
 image::git/git-commit-tree-view.png[]
 
-You can select files which will be added to index and committed. All files in the selected package / folder in the project explorer will be checked by default. After selecting files, type commit message. Optionally you could select `Amend previous commit` to replace previous commit (for more details https://git-scm.com/docs/git-commit#git-commit---amend[see git docs]).
+Select the files that you want to add to the index and commit. Note that all the files in the selected package or folder in the *Project Explorer* are selected by default. Type the commit message. Click *Amend previous commit* to replace the previous commit. For more details, see https://git-scm.com/docs/git-commit#git-commit---amend[git docs].
 
-Push your commit to a remote repository by checking `Push committed changes to` check-box and selecting a remote branch.
+To push your commit to a remote repository, click *Push committed changes to* and select a remote branch.
 
-Click `Commit` to proceed (the `Commit` button is active as long as at least 1 file is selected and a commit message is present or `Amend previous commit` is checked).
+Click *Commit* to proceed. The *Commit* button is active if at least one file is selected and a commit message is present or *Amend previous commit* is selected.
 
-Behavior for files in the list view is the same as in compare window (see link:#compare[Reviewing changed files] section). Double clicking a file will open diff window with it.
+Behavior of the files in the list view is the same as in the compare window (see the link:#compare[Reviewing changed files] section). Double-click a file to open the diff window with it.
 
-*Push/Pull*
+=== Push/Pull
 
-Push your commits at `Git` -> `Remotes...` -> `Push` from main menu or `Alt + Shift + c` shortcut.
+Push your commits using the *Git* > *Remotes* > *Push* menu options or use the `Alt + Shift + c` keyboard shortcut.
 
 image::git/git-push.png[]
 
-Choose a remote repository, local and remote branch. Optionally, you can force `Force push`. Get changes from a remote repo at `Git` -> `Remotes...` -> `Pull` (`Alt + P` shortcut).
+Choose the remote repository, the local branch, and the remote branch. Optionally, you can use `Force push`. To get changes from a remote repository, use the *Git* > *Remotes* > *Pull* or use the `Alt + P` keyboard shortcut.
 
 image::git/git-pull.png[]
 
-You can use `Rebase instead of merge` to keep your local commits on top (for more information https://git-scm.com/docs/git-pull#git-pull--r[see git docs]).
+Use `Rebase instead of merge` to keep your local commits on top. For more information, see the https://git-scm.com/docs/git-pull#git-pull--r[git docs].
```

Comment:
"`Rebase instead of merge`" is a UI element -- shouldn't this be bold instead of monospace?